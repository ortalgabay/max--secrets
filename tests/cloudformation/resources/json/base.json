{
  "AWSTemplateFormatVersion": "2010-09-09",
  "Resources": {
    "S3Bucket": {
      "Type": "AWS::S3::Bucket",
      "Properties": {
        "AccessControl": "PublicRead",
        "WebsiteConfiguration": {
          "IndexDocument": "index.html",
          "ErrorDocument": "error.html"
        },
        "Tags": [
          {
            "Key": "old_tag",
            "Value": "old_value"
          },
          {
            "Key": "yor_trace",
            "Value": "08d850c8-a4fc-4d1c-9851-bb3a72b2674b"
          }
        ]
      },
      "DeletionPolicy": "Retain"
    }
  }
}