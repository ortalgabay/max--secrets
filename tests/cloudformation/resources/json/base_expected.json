{
  "AWSTemplateFormatVersion": "2010-09-09",
  "Resources": {
    "S3Bucket": {
      "Type": "AWS::S3::Bucket",
      "Properties": {
        "AccessControl": "PublicRead",
        "WebsiteConfiguration": {
          "IndexDocument": "index.html",
          "ErrorDocument": "error.html"
        },
        "Tags": [
          {
            "Key": "old_tag",
            "Value": "old_value"
          },
          {
            "Key": "new_tag",
            "Value": "new_value"
          },
          {
            "Key": "another_tag",
            "Value": "another_val"
          },
          {
            "Key": "yor_trace",
            "Value": "8dd59656-404a-47c4-8136-5e165622b5b2"
          }
        ]
      },
      "DeletionPolicy": "Retain"
    }
  }
}